nmigen_boards/atlys.py

```diff
@@ -0,0 +1,235 @@
+import subprocess
+from textwrap import dedent
+
+from nmigen.build import *
+from nmigen.vendor.xilinx_spartan6 import *
```

Comment:
Needs to be changed to `nmigen.vendor.xilinx_spartan_3_6` in light of [Spartan 3A](https://github.com/m-labs/nmigen/pull/138) support being merged in `nmigen`.